hy/core/macros.hy

```diff
@@ -276,3 +276,10 @@ Such 'o!' params are available within `body` as the equivalent 'g!' symbol."
                 _hy_tag
                 [None]
                 ['~symbol])))))
+
+
+(defmacro pun [&rest body]
+  (HyExpression (reduce + (gfor x body
+    (if (and (keyword? x) (.startswith x.name "^"))
+      [(HyKeyword (cut x.name 1)) (HySymbol (cut x.name 1))]
+      [x])))))
```

Comment:
This definition isn't indented properly. Make sure it passes parlinter.

Reply:
[I asked you to stop this.](https://github.com/hylang/hy/pull/1338#discussion_r131307803) [Twice.](https://github.com/hylang/hy/pull/1378#discussion_r132749045) [Then you insulted me about it.](https://github.com/hylang/hy/pull/1614#issuecomment-391211451) Please don't comment on my indentation again.